Common/Charting.cs

```diff
@@ -334,4 +391,61 @@ public enum ChartType
         /// Stacked series on top of each other.
         Stacked
     }
+
+    /// <summary>
+    /// Color to the series object
+    /// Basic colors
+    /// </summary>
+    public enum SeriesColor
```

Comment:
I would prefer we use the build in `Color` struct for .NET. It is an open type and allows specifying individual RGB components as well as specifying colors directly, such as `Color.Blue`. In addition, the `Color` struct has some strange serialization rules. If it is a 'KnownColor' then it will serialize a string of the color's name, if it is not a known color then it will serialize comma separated RGB values. Let's not aim to support A (alpha) at the moment.

We can modify the serialization by creating a custom `JsonConverter`. For ease of use in the front-end, we desired the output of serialization to be of the form: `rgb(r,g,b)` so for a `Color` with `Red=123; Green=231; Blue=067` we expect an output of: `rgb(123,231,67)`. I would derive from the `TypeChangeJsonConverter<Color, string>` to make this a little simpler.